src/app/shared/ag-grid-renderers/percentage-renderer/percentage-renderer.component.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  {{value}}%
```

Comment:
We should not create a renderer just to show a percentage sign, please look into other alternatives and discuss with me